src/licensedcode/data/rules/false-positive_772.RULE

```diff
@@ -0,0 +1,14 @@
+---
+is_false_positive: yes
+notes: seen in kernel /Documentation/process/license-rules.rst
+---
+
+The SPDX license identifier is added in form of a comment.  The comment
```

Comment:
What about having multiple smaller, less specific FP rules for all the related FPs in this PR, here and below?
- SPDX license identifier is added
- SPDX-License-Identifier: <SPDX License
- (SPDX-License-Identifier) nor in
- SPDX license identifier in